mitiq/zne/scaling/parameter.py

```diff
@@ -31,17 +31,16 @@
 from mitiq.conversions import converter
 
 
-BASE_GATES = [ZPowGate, HPowGate, XPowGate, YPowGate, CXPowGate, CZPowGate]
-
-
 class GateTypeException(Exception):
     pass
 
 
 def _get_base_gate(gate: EigenGate) -> EigenGate:
```

Comment:
Is there any case where something is done to the input gate that is not checking membership in the set of base gates? If not, I would suggest to call this `_check_base_gate` or `_check_supported_gate` and simplify this to just:

```python
BASE_GATES = [ZPowGate, HPowGate, XPowGate, YPowGate, CXPowGate, CZPowGate]


def _check_supported_gate(gate: EigenGate) -> None:
    if gate not in BASE_GATES:
        <raise error>
```

Reply:
yeah we actually want to return the superclass of the gate. so we just get in a vanilla X gate, we want to return the XPowGate, not say that the X gate is ok